include/nlohmann/detail/output/serializer.hpp

```diff
@@ -236,6 +237,50 @@ class serializer
                 return;
             }
 
+            case value_t::binary:
```

Comment:
We may want to control this behavior similar to the `error_handler_t` in https://github.com/nlohmann/json#character-encoding. Just going for a string like this should not be the only option.

Reply:
I'm attempting to do this by adding a parameter, just like the character encoding.  However, the testing harness simply calls `operator<<`, it doesn't call dump directly, and I don't want to override `operator<<`'s just to make this call work.  I can adjust the reaction to some `iomanip` flags, but I feel like that would have unintended consequences or be otherwise surprising, and I do not want to add a bool to the json data structure to store just this toggle, seems inelegant.  Any ideas?

Reply:
`public static` member to toggle it?

Reply:
For now, I'm using a public static member.  This isn't very nice because it introduces global state into serialization.  And I'm open to other ideas.

Reply:
Where is that static member?

Reply:
(Forget it - I found it.)

Reply:
`operator<<` can only be controlled in a very limited way - right now, only the level of indentation can be controlled via `std::setw`. This is fine, because it is only a convenience way of calling the `dump` function with more detailed parameters. We should not add any API just to allow all options from `dump` to also work with `operator<<`.